aiplatform/api/AIPlatform.Samples/PredictTextSentimentSample.cs

```diff
@@ -0,0 +1,115 @@
+(the actors deliver most of their lines directly to the camera) to the camera
+shots (all medium or close up shots...no scenery shots and very little back
+ground in the shots) were absolutely terrible. I watched this over my spring
+break and it is very safe to say that I feel that I was gypped out of 114
+minutes of my vacation. Not recommended by any stretch of the imagination.
+Classify the sentiment of the message: negative
+
+Something surprised me about this movie - it was actually original. It was
+not the same old recycled crap that comes out of Hollywood every month. I saw
+this movie on video because I did not even know about it before I saw it at my
+local video store. If you see this movie available - rent it - you will not
+regret it.
+Classify the sentiment of the message: positive
+
+My family has watched Arthur Bach stumble and stammer since the movie first
+came out. We have most lines memorized. I watched it two weeks ago and still
+get tickled at the simple humor and view-at-life that Dudley Moore portrays.
+Liza Minelli did a wonderful job as the side kick - though I'm not her
+biggest fan. This movie makes me just enjoy watching movies. My favorite scene
+is when Arthur is visiting his fiancée's house. His conversation with the
+butler and Susan's father is side-spitting. The line from the butler,
+""Would you care to wait in the Library"" followed by Arthur's reply,
+""Yes I would, the bathroom is out of the question"" is my NEWMAIL
+notification on my computer.
+Classify the sentiment of the message: positive
+
+This Charles outing is decent but this is a pretty low-key performance. Marlon
+Brando stands out. There's a subplot with Mira Sorvino and Donald Sutherland
+that forgets to develop and it hurts the film a little. I'm still trying to
+figure out why Charlie want to change his name.
+Classify the sentiment of the message: negative
+
+Tweet: The Pixel 7 Pro, is too big to fit in my jeans pocket, so I bought new
+jeans.
+Classify the sentiment of the message: ";
+
+        var instances = new List<wkt::Value>
+        {
+            wkt::Value.ForStruct(new()
+            {
+                Fields =
+                {
+                    ["content"] = wkt::Value.ForString(content),
+                }
+            })
+        };
+
+        var parameters = wkt::Value.ForStruct(new()
+        {
+            Fields =
+            {
+                { "temperature", new wkt::Value { NumberValue = 0 } },
+                { "maxDecodeSteps", new wkt::Value { NumberValue = 5 } },
+                { "topP", new wkt::Value { NumberValue = 0 } },
+                { "topK", new wkt::Value { NumberValue = 1 } }
+            }
+        });
+
+        // Make the request.
+        var response = client.Predict(endpoint, instances, parameters);
+
+        // Parse and return the content.
+        var responseContent = response.Predictions[0].StructValue.Fields["content"].StringValue;
```

Comment:
Technically, you could have multiple predictions, that's why it's an array, but in all my runs, I've never seen it more than 1. Here and in all other samples, we're fetching the first and the only prediction.

Reply:
@meteatamel but if there were to be multiple results, would the first one be the better model answer? That's a valid question that users may ask themselves, and it would be good to have an answer for them, either in code or in the docs where the sample is shown.

Is there's scoring per answer, then you probably want to show how to sort the answers on score on return the highest scoring answer. If really there's always one single answer, and we have AI Platform API docs that say so, then we should explain that in the sample docs and link ot the API docs.